docs/releaseNotes.yml

```diff
@@ -36,16 +36,23 @@ items:
     prevVersion: 3.4.0
     date: 'TBD'
     notes:
-      - title: Add support for startupProbe on the emissary-ingress deployments
-        type: change
+      - title: Fix envoy config generation when including port in Host.hostname
```

Comment:
I took it out because it is a Helm only change and doesn't really change behavior of emissary itself. This is probably inline with our discussion on Friday about how do we want to handle helm changes in the docs release notes vs repository release notes.

My thought is that this belongs on the change log for Helm and then when we do our release docs we can include a catch-all item for Helm changes, thoughts?